pommerman/envs/utility.py

```diff
@@ -52,6 +52,8 @@ class GameType(Enum):
     Team = 2
     # 2v2: You submit two agents. They are matched up against two other agents. Each team passes discrete communications to each other.
     TeamRadio = 3
+    # 2v2: Game ends when any player dies. Team with most living players still alive wins.
+    TeamSuddenDeath = 4
```

Comment:
I don't fully understand, is this new mode like `Team` or like `TeamRadio`? And wouldn't we want a "sudden death" mode for both?

Reply:
This is for agent training purposes. I intend to train separate DQNs for 2v2 play and 2v1 play. This mode would allow for faster training of the 2v2. Others can benefit from this game mode existing.

Reply:
The GameType is meant to represent the types of games that are offered in Battles and Competitions. Sudden death is something that could be applied to any of them, and so I think you should make a new environment with this behavior.

An example of this can be found in v1.py where the collapsing walls is added to v0.py. This could be v3.py and inherit from v0.py.